.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
eslint-config-cozy-app is supposed to bring this, no?

Reply:
That's file is for the editor if I understand well. @CPatchane could you confirm please ?

Reply:
Indeed, we need this file to handle correctly prettier in Atom (except if you use eslint-prettier I think).